pallets/payment-streams/src/lib.rs

```diff
@@ -486,8 +494,10 @@ pub mod pallet {
             // Check that the extrinsic was executed by the root origin
             ensure_root(origin)?;
 
+            let current_tick = Self::get_current_tick();
+
             // Execute checks and logic, update storage
-            Self::do_delete_fixed_rate_payment_stream(&provider_id, &user_account)?;
+            Self::do_delete_fixed_rate_payment_stream(&provider_id, &user_account, current_tick)?;
```

Comment:
`do_delete_fixed_rate_payment_stream` can lookup the `current_tick` on its own, it doesn't need to receive it as a parameter.